setup.py

```diff
@@ -139,16 +139,15 @@ def run_setup():
         include_package_data=True,
```

Comment:
In setup.py you could *strictly* do a check against python version and set requirements based on that and pass through. Still not sure that's wise, but worth noting.

Reply:
I don't think it's necessary here. It's a useful tool when things go wrong, but for now let's assume that the packages we are using all define their requirements correctly and make the installer responsible for getting a compatible set.

If we hit trouble with a dependency's metadata, that's the time to introduce special version-dependent pins.